auth/README.md

```diff
@@ -0,0 +1,131 @@
+## Overview
+
+This implements authentication and authorization for Veraison API.
+Authentication can be performed using the Basic HTTP scheme (with the `basic`
+backend), or using a Bearer token (with the `keycloak` backend). Once an API
+user is authenticated, authorization is role-based. See documentation for
```

Comment:
```suggestion
user is authenticated, authorization is role-based. See documentation [here](link to documentation) for
```

Reply:
There isn't  really a link here -- different services have different documentation, and trying to maintain a comprehensive set of links here is going to do more harm than good, as it is pretty much guaranteed to get out of sync.